### ISSUE TITLE
Start coding the frontend

### ISSUE THREAD

**Issue by gabriel-lody** [lodysoft/roubamonte#3]
Remember to create a new branch before commiting the frontend code.

**Comment by gabriel-lody** [comment]
At the end I thought we decided for SDL2 since the beginning, but it's sfml it could be ok for a test, at least. The only issue I'm seeing is we're using Make, not CMake. Check the `test_suite` directory, it has a very beautiful `Makefile`. Be careful with Claude, you should guide him, not the other way around. My fight is always forcing him to stick to good practices. He has a tendency to do what most people do and that's not desirable for this project.

**Comment by appeye** [comment]
yeah, creating a new branch `frontend-sdl2`  to do the port sfml -> sdl2

**Comment by gabriel-lody** [comment]
Perfect! Don't forget we're using Make instead of CMake. Thank you so much!
